### PR TITLE
docs: fixing the icons in pfe-navigation

### LINKED ISSUE #700: pfe-navigation icons are broken in Storybook
Here's how the icons look in Storybook:
image

Here's a screenshot of the code:
image

Note the "load-failed" class indicating that the icon could not be found.

### PR DESCRIPTION
**Pull request by kylebuch8**
Fixes #700

### CODE REVIEW

File: elements/pfe-navigation/demo/pfe-navigation.story.js [hunk 1]
```diff
@@ -92,7 +92,7 @@ stories.add(PfeNavigation.tag, () => {
     })
   }) : "";
 
-  let search = slotCheck.search ? createItem("search", "search", "Search", `<div class="pfe-navigation-item__tray--container">
+  let search = slotCheck.search ? createItem("search", "web-search", "Search", `<div class="pfe-navigation-item__tray--container">
```
Comment (starryeyez024) [inline elements/pfe-navigation/demo/pfe-navigation.story.js]: I don't understand what this is doing, in master the icon that is loaded is:
`https://access.redhat.com/webassets/avalon/j/lib/rh-iconfont-svgs/web-icon-search.svg` 
which is correct

Comment (kylebuch8) [reply elements/pfe-navigation/demo/pfe-navigation.story.js]: All of our icons are prefixed with something and then have a dash. The prefix is the set that the icon is coming from. We have a set called "web" that has the Red Hat web icons. Using `search` doesn't tell `pfe-icon` which set the icon is coming from.

Comment (kylebuch8) [reply elements/pfe-navigation/demo/pfe-navigation.story.js]: @starryeyez024 These are the built-in icons sets that we have.

https://github.com/patternfly/patternfly-elements/blob/master/elements/pfe-icon/src/builtin-icon-sets.js#L16

Comment (starryeyez024) [reply elements/pfe-navigation/demo/pfe-navigation.story.js]: Sorry, I dont understand how it is currently broken in master, it seems to be working already

Comment (kylebuch8) [reply elements/pfe-navigation/demo/pfe-navigation.story.js]: The icons in the story are not working in master. This fixes it.

Comment (kylebuch8) [reply elements/pfe-navigation/demo/pfe-navigation.story.js]: Maybe taking a quick look at the issue might help.
https://github.com/patternfly/patternfly-elements/issues/700

File: elements/pfe-navigation/README.md [hunk 2]
```diff
@@ -6,14 +6,59 @@ This element will give you a top-level navigation element (not for use as subnav
...
+    </div>
+    ```
+
+- `logo`: This slot should contain the site's main logo. Note that there is also a CSS variable for the width of the logo, `--pfe-navigation__logo--MinWidth`. Example: 
+
+    ```
+    <a style="--pfe-navigation__logo--MinWidth: 160px" slot="logo" href="https://company.com"><img class="logo" src="https://via.placeholder.com/150x50.png" title="Company logo" /></a>
+    ```
+
+- `search`: This slot, when assigned to a navigation-item component, renders a dropdown in the navigation for the search functionality. Inside the tray for the slot, we recommend tagging the search form or search functionality that includes the input and submit button with the attribute `pfe-navigation--mobile-search`. The element inside the search slot that is tagged with this attribute will be copied into the mobile menu and appear above the accordion for the main navigation. Example:
```
Comment (castastrophe) [inline elements/pfe-navigation/README.md]: @kylebuch8 Do we know if search and language work without being assigned to pfe-navigation-item?  I would guess it does but haven't tested that.

File: elements/pfe-navigation/README.md [hunk 3]
```diff
@@ -6,14 +6,59 @@ This element will give you a top-level navigation element (not for use as subnav
 
 ### Slots
```
Comment (castastrophe) [inline elements/pfe-navigation/README.md]: Suggest adding this note at the top of the list:

All slots are optional and can be left off if not needed.  Slots prefixed with `mobile-`, however, do require their counterparts in order to render.

File: elements/pfe-navigation/README.md (hunk 2 above)
Comment (kylebuch8) [reply elements/pfe-navigation/README.md]: @castastrophe Can we create a new issue for README updates? I'd like to keep this PR aligned to #700 that it's fixing.

Comment (castastrophe) [reply elements/pfe-navigation/README.md]: Sure, was only commenting on it because it was updated in this PR.

Comment (kylebuch8) [reply elements/pfe-navigation/README.md]: @castastrophe HA! I had no idea @starryeyez024 piled some commits on to this.

**Pull request merged**